Pod/Classes/SwiftPhotoGallery.swift

```diff
@@ -143,11 +143,20 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
 
 
     // MARK: UICollectionViewDelegate Methods
+
+    public func scrollViewWillBeginDragging(scrollView: UIScrollView) {
+        pageControl.alpha = 1.0
```

Comment:
Changed the bottom constraint constant to a variable which gets changed by the animation. This doesn't seem to update the constraint however.